### PR TITLE
dashboard: Show processes diff in confirmation dialog when scaling

### PR DESCRIPTION
**Pull request by jvatic**
screen shot 2018-06-12 at 11 20 35

Also updated the dashboard-dev script (related: #1452)

### CODE REVIEW

**Comment by titanous** [comment]
typo in commit message: "dashbaord"

File: dashboard/bindata.go [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 
 func Asset(path string) ([]byte, error) {
-	return nil, nil
+    file, err := os.Open(path)
+    if err != nil {
+        return nil, err
+    }
+    var buf bytes.Buffer
+    _, err = io.Copy(&buf, file)
+    return buf.Bytes(), err
```
Comment (titanous) [inline dashboard/bindata.go]: Use `ioutil.ReadFile`

File: script/dashboard-dev [hunk 2]
```diff
@@ -55,27 +60,20 @@ run() {
...
-    _, err = io.Copy(&buf, file)
-    return buf.Bytes(), err
+	return ioutil.ReadFile(path)
 }
 
 func AssetInfo(path string) (os.FileInfo, error) {
-    return os.Stat(path)
+	return os.Stat(path)
 }
 BINDATA
```
Comment (jvatic) [inline script/dashboard-dev]: We possibly don't need this. I'll double check and see if any of the compile steps overwrite the checked-in version of `bindata.go` (this was previously the case).

Comment (jvatic) [reply script/dashboard-dev]: Confirmed. This isn't needed.

File: dashboard/api_test.go [hunk 3]
```diff
@@ -22,16 +22,19 @@ const (
 func Test(t *testing.T) { TestingT(t) }
 
 type S struct {
-	srv *httptest.Server
+	srv        *httptest.Server
+	cookiePath string
```
Comment (titanous) [inline dashboard/api_test.go]: This looks unused?

Comment (jvatic) [reply dashboard/api_test.go]: Oops, supposed to be used down in one the assertion for the Location header. I'll create another PR.

Comment (jvatic) [reply dashboard/api_test.go]: https://github.com/flynn/flynn/pull/4403

**Pull request merged**